### ISSUE TITLE
how to hide the ScrollContainer's scrollbar.My Godot version is 3.0.1

### ISSUE THREAD

**Issue by chenjie199234** [godotengine/godot#17165]
dont know how to hide the scrollcontainer's scrollbar.

**Comment by akien-mga** [comment]
Please ask such questions on the [Q&A](https://godotengine.org/qa) or other [community channels](https://godotengine.org/community).

The GitHub issue tracker is for bug reports, not support.

Regarding your question, see the docs: http://docs.godotengine.org/en/latest/classes/class_scrollcontainer.html

**Comment by TimurKhan** [comment]
You know, I am sometimes surprised by the position of the local "masters". Without knowing the answers to the question, it is very convenient to refer to the documentation.
There are no answers there!

For example, I cannot "extinguish" the scrollbar in the tree control. No way! I've tried setting visibility to 0, I've tried doing it from _process. Show me thet place in the documentation where I missed how to hide tree scrollbar?
